src/dev-core/src/app/icon-sets/icon-sets.demo.html

```diff
@@ -1,133 +1,119 @@
 <style>
-    table.table {
-        width: 100%;
-        max-width: 32rem;
-        margin: 1.2rem 0;
-    }
+  table.table {
+      width: 100%;
+      max-width: 32rem;
+      margin: 1.2rem 0;
+  }
 
-    table.table th,
-    table.table td {
-        text-align: left;
-        line-height: 1.1em;
-    }
+  table.table th,
+  table.table td {
+      text-align: left;
+      line-height: 1.1em;
+  }
 
-    .style-toggles label,
-    p > label {
-        padding-left: 0.9rem;
-    }
-    
-    .style-toggles label:first-child,
-    p > label:first-child {
-        padding-left: 0;
-    }
+  .style-toggles label,
+  p > label {
+      padding-left: 0.9rem;
+  }
+  
+  .style-toggles label:first-child,
+  p > label:first-child {
+      padding-left: 0;
+  }
 
-    .filter-and-toggles {
-        border-bottom: 0.05rem solid #ccc;
-        padding-bottom: 1.2rem;
-        margin-bottom: 1.2rem;
-    }
-    
-    table b {
-        padding-right: 0.6rem;
-    }
-    tr + tr > td {
-        padding-top: 0.6rem;
-    }
+  .filter-and-toggles {
+      border-bottom: 0.05rem solid #ccc;
+      padding-bottom: 1.2rem;
+      margin-bottom: 1.2rem;
+  }
 
-    .style-toggles input {
-        padding-right: 0.4rem;
-    }
+  table b {
+      padding-right: 0.6rem;
+  }
+  tr + tr > td {
+      padding-top: 0.6rem;
+  }
 
-    .dc-icon-set cwc-icon {
-        width: 1.8rem;
-        height: 1.8rem;
-    }
+  .style-toggles input {
+      padding-right: 0.4rem;
+  }
 
-    .dc-icon-box table {
-        margin: 0;
-    }
-    
-    .dc-icon-box table td {
-        text-align: center;
-    }
+  .dc-icon-set cwc-icon {
+      width: 1.8rem;
+      height: 1.8rem;
+      margin-bottom: 0.4rem;
```

Comment:
Using the `div`s and `p`s the way they are being used makes the space between the icon and the label blow out. Plz remove this `margin-bottom` declaration so that the layout remains consistent.